src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -213,7 +209,7 @@ internal sealed override ImmutableArray<string> NotNullWhenFalseMembers
                    isIterator)
         {
             _property = property;
-            _isAutoPropertyAccessor = isAutoPropertyAccessor;
+            _accessorBodyShouldBeSynthesized = property.IsAutoProperty && syntax is AccessorDeclarationSyntax { SemicolonToken.RawKind: (int)SyntaxKind.SemicolonToken, ExpressionBody: null, Body: null };
```

Comment:
@AlekseyTs I'm still asking the property a question, but the question has method from `MethodCompiler` to `SourcePropertyAccessorSymbol`. Is that the kind of refactor you want? I can't think of a way to completely avoid asking the property.

Reply:
Actually I think this will crash, should have used `IsAutoPropertyWithGetAccessor`, or the other solution is to handle null backing field in `ConstructAutoPropertyAccessorBody`. Let me know what you prefer